src/services/initial-products.js

```diff
@@ -0,0 +1,13 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+export const initialProducts = () => {
+  const [initialProductList, setInitialProductList] = useState([]);
+
+  useEffect(async () => {
+    const data = await axios.get("/api/products");
+    setInitialProductList(() => data.data.products);
+  }, []);
```

Comment:
liked that you have created a separate file for products. you can destructure data here